php/commands/role.php

```diff
@@ -367,6 +383,167 @@ private static function persistence_check() {
 		if ( !$wp_roles->use_db )
 			WP_CLI::error( "Role definitions are not persistent." );
 	}
+
+	private static function get_role_capability( $role_key ) {
```

Comment:
@rahulsprajapati I'm still hesitant about internalizing this array of roles and capabilities because I'm concerned it will likely get out of date.

Have you thought of any other ways to produce the same effect? If not, I think we might want to punt on this.